requirements.txt

```diff
@@ -12,7 +12,7 @@ pycryptodomex==3.17
 PyJWT==2.6.0
 pyOpenSSL==23.0.0
 pytz==2022.7.1
-requests==2.28.2
+requests==2.31.0
```

Comment:
Updated to match the version on platform and freeze it at this version (requests cannot be completely deleted from the connector because it is installed as a dependency of snowflake package)

Reply:
Maybe add this info to the file as a short comment?

Reply:
Good idea, added.